lib/component-repository/src/routes/virtual-components/index.js

```diff
@@ -95,11 +94,11 @@ module.exports = ({ iam }) => {
   );
 
   // To Implement
-  router.get('/:id/:componentVersionId', GetComponentVersion);
-  router.post('/:id/:componentVersionId/config', CreateComponentConfig);
-  router.patch('/:id/:componentVersionId/config', UpdateComponentConfig);
-  router.get('/:id/:componentVersionId/actions/:actionName', GetAction);
-  router.get('/:id/:componentVersionId/triggers/:triggerName', GetTrigger);
+  router.get('/:id/:componentVersionId',loadVirtualComp, GetComponentVersion);
+  router.post('/:id/:componentVersionId/config',loadVirtualComp, bodyParser, CreateComponentConfig);
+  router.patch('/:id/:componentVersionId/config',loadVirtualComp, bodyParser, UpdateComponentConfig);
```

Comment:
Add the corresponding permission like:
can(config.componentsCreatePermission) or update...